kernel/kernel.h

```diff
@@ -114,13 +114,29 @@ int printf(const char *fmt, ...);
 int snprintf(char *str, size_t size, const char *format, ...);
 int vsnprintf(char *buf, size_t size, const char *fmt, va_list args);
 
+/* quiet < error < warning < info < debug */
+enum log_level {
+    QUIET=0,
```

Comment:
I see no point in disabling _all_ messages (including `ERROR` which are critical errors), so `QUIET` can go away.